br/pkg/lightning/mydump/parquet_parser.go

```diff
@@ -566,3 +576,14 @@ func (*ParquetParser) SetColumns(_ []string) {
 func (pp *ParquetParser) SetLogger(l log.Logger) {
 	pp.logger = l
 }
+
+func jdToTime(jd int32, nsec int64) time.Time {
+	sec := int64(jd-jan011970) * secPerDay
+	return time.Unix(sec, int64(nsec))
```

Comment:
```suggestion
	return time.Unix(sec, nsec)
```

And can you add unit tests for nsec outside [0, 999,999,999]? (see the comment of `time.Unix`)

Reply:
Will take a look that this later.

Reply:
2536ce5